manifests/package/debian.pp

```diff
@@ -40,13 +41,15 @@
           location => "https://nginx.org/packages/${distro}",
           repos    => 'nginx',
           key      => {'id' => '573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62'},
+          release  => pick($release, $::lsbdistcodename),
```

Comment:
You can actually pass ```undef``` to puppetlabs-apt and let the module handle it meaning you don't need to pick: using ```$release``` will just work.

Reply:
Good point, changed in https://github.com/voxpupuli/puppet-nginx/pull/1154/commits/b85c0607cfc0081edbde4f48496b543480004667